Pod/Classes/PURBufferedOutput.m

```diff
@@ -50,11 +50,16 @@ - (void)dealloc
 - (void)setUpTimer
 {
     [self.timer invalidate];
-    self.timer = [NSTimer timerWithTimeInterval:1.0
-                                         target:self
-                                       selector:@selector(tick)
-                                       userInfo:nil
-                                        repeats:YES];
+    if(self.flushInterval <= 0){
```

Comment:
Why this protection for the flush interval? Are we expecting it to be non-positive at some point?

Reply:
This setting is optional, in case that is not configured, needs to be protected to set a valid value.

Reply:
Fair enough.